### PR TITLE
Bug fix and maintenance quest

### PR DESCRIPTION
**Pull request by Nekotekina**
Bug may be related to #6532

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellKb.cpp [hunk 1]
```diff
@@ -33,15 +33,11 @@ error_code cellKbInit(u32 max_connect)
 {
 	sys_io.warning("cellKbInit(max_connect=%d)", max_connect);
 
-	auto handler = fxm::get<KeyboardHandlerBase>();
-
-	if (handler)
-		return CELL_KB_ERROR_ALREADY_INITIALIZED;
+	const auto handler = g_fxo->get<KeyboardHandlerBase>();
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/Modules/cellKb.cpp]: Wtf is this

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellKb.cpp]: Missing error checking..

File: rpcs3/Emu/Cell/Modules/cellCamera.cpp [hunk 2]
```diff
@@ -414,8 +418,12 @@ s32 cellCameraOpenEx(s32 dev_num, vm::ptr<CellCameraInfoEx> info)
 		return CELL_CAMERA_ERROR_PARAM;
 	}
 
-	const auto g_camera = fxm::get<camera_thread>();
-	// we know g_camera is valid here (cellCameraSetAttribute above checks for it)
+	const auto g_camera = g_fxo->get<camera_thread>();
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/Modules/cellCamera.cpp]: What about the comment above?

File: rpcs3/Emu/Cell/Modules/cellKb.cpp (hunk 1 above)
Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellKb.cpp]: The whole commit is gonna break the modules

File: rpcs3/Emu/Cell/Modules/cellCamera.cpp [hunk 3]
```diff
@@ -277,12 +277,7 @@ s32 cellCameraInit()
 	cellCamera.todo("cellCameraInit()");
 
 	// Start camera thread
-	const auto g_camera = fxm::make<camera_thread>("Camera Thread");
-
-	if (!g_camera)
-	{
-		return CELL_CAMERA_ERROR_ALREADY_INIT;
-	}
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/Modules/cellCamera.cpp]: Idk if moving this check down is correct for null handler. The games still need to call the functions in correct order

**Pull request merged**